packages/hint-font-relative-units/src/hint.ts

```diff
@@ -0,0 +1,90 @@
+/**
+ * @fileoverview Ensure font styles use relative units to enable user stylesheet overrides
+ */
+
+import { Declaration, Rule } from 'postcss';
+import { HintContext } from 'hint/dist/src/lib/hint-context';
+import { IHint, ProblemLocation } from 'hint/dist/src/lib/types';
+import { debug as d } from '@hint/utils';
+import { getCSSCodeSnippet } from '@hint/utils/dist/src/report/get-css-code-snippet';
+import { StyleEvents, StyleParse } from '@hint/parser-css';
+
+import meta from './meta';
+import { getMessage } from './i18n.import';
+
+const debug: debug.IDebugger = d(__filename);
+
+/*
+ * ------------------------------------------------------------------------------
+ * Public
+ * ------------------------------------------------------------------------------
+ */
+
+export default class FontA11yFontSizeUnitsHint implements IHint {
+
+    public static readonly meta = meta;
+    public readonly FONT_PROPERTIES = ['font-size', 'line-height', 'letter-spacing'];
+
+    public constructor(context: HintContext<StyleEvents>) {
+
+        const getLocation = (decl: Declaration): ProblemLocation => {
+            const start = decl.source && decl.source.start;
+
+            if (start) {
+                return {
+                    column: start.column - 1,
+                    line: start.line - 1
+                };
+            }
+
+            return {
+                column: 0,
+                line: 0
+            };
+        };
+
+        const validateFontValue = (prop: string, value: string): boolean => {
+            if (this.FONT_PROPERTIES.includes(prop)) {
+                return value.indexOf('px') === -1;
```

Comment:
I think you need to check also the viewport units per this comment in the hint description:

>viewport units aren’t relative units. I know they’re listed under relative units and described as such, but they don’t always act like relative units (resizing font sizes set with viewport units worked for me in Safari but not in Chrome and Firefox).